frontend/src/Navbar.js

```diff
@@ -0,0 +1,23 @@
+import {Link, useMatch, useResolvedPath} from "react-router-dom"
+export default function Navbar() {
+    return (
+    <nav className="nav">
```

Comment:
Name class something else, even "navStyle" or something is fine. This is just because you can actually edit the CSS for a nav tag by default, so it might be confusing if later you want to have a style for nav and a style for .nav. This is kind of just nitpick